handbook/company/asynchronous-communication.md

```diff
@@ -0,0 +1,152 @@
+- **Never assume.** It is best to include more context than to give room to misinterpretation. 
+- **Share**. Don’t make others hunt for information.
+- **Communicate your work preferences.** We all work differently and have preferred ways of communication. Let your teammates know, this will help everyone be more empathetic and conscious about collaboration.: Create a rudimentary README that clarifies how you work. You can find some great examples [here](https://about.sourcegraph.com/handbook/support/support-bios).
+- **Summarize**. If your documentation ends up being too long, consider adding a summary, in the beginning, to help people consider if this is what they are looking for.
+- **Consider which tool to use**. Think about what you are working on and which is the best tool to discuss it, iterate and communicate.
+- **Know when *not* to meet**. Ask yourself: is this meeting necessary or could we work on it asynchronously?
+- **Don’t be afraid to cancel or reject meetings**. Hopping on to meetings is easy and we tend to find it easy, however, they are also very disrupting and demanding. It’s completely okay to ask someone to cancel a meeting and use that time to communicate asyncly what they wanted to share with you. You’ll find some time to come back to them with your opinion and ideas.
+- **Meetings must start and end on time**. We must be conscious of our and other people’s time.
+- **Innovate**. Try out new things. Talk to your team and let them know you’ll conduct an asynchronous pilot. You won’t assist all or some of your weekly meetings and come back to them at the end of the week with your discoveries. Share them with the team and think of ways to make that experience better in the future.
+- **Invest in your writing**. It takes more time, but it gives people time to read/think about what you said versus react immediately and it is scalable.
+- **Set timing expectations**. Add parameters like response time and due dates to ensure progress.
+- **Give yourself time**. Give yourself time to adjust to this new way of working - and to do this type of work. Thoughtfully crafting, reading, and responding to async communication takes longer (in the upfront) than the rapid back-and-forth of real-time conversation. 
+- **Synchronous meetings.** Consider synchronous meetings when needed. Synchronous meetings are not the enemy and can provide a ton of value. Make sure to be inclusive and respectful of other people's time when you schedule them.
+
+Each team should also have a section to explain how they do x, y and z asyncly. Define processes and communication channels. For example, how does the design team work? What is the decision making process? 
+
+
+## How to choose sync vs async?
+
+<img width="60%" height="60%" src="onepager-async.jpg">
```

Comment:
Is there a chance this could be exported as an SVG instead of JPG? It has a lot of text content that won't be readable for screen reader users (but would be if it was an SVG)
What design tool was it created in?